frameworks/digital-outcomes-and-specialists-5/manifests/display_brief.yml

```diff
@@ -59,6 +59,7 @@
     - niceToHaveRequirementsParticipants
 -
   name: How suppliers will be evaluated
+  summary_page_description: "All suppliers will be asked to provide a {% if lotSlug == 'digital-specialists' %}work history{% else %}written proposal{% endif %}."
```

Comment:
When testing in flask shell I get the error 

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/Users/laurencedebruxelles/Code/digitalmarketplace/digitalmarketplace-content-loader/dmcontent/content_loader.py", line 192, in __getattribute__
    return field.render(context)
  File "/Users/laurencedebruxelles/Code/digitalmarketplace/digitalmarketplace-content-loader/dmcontent/utils.py", line 66, in render
    raise ContentTemplateError(e.message)
dmcontent.errors.ContentTemplateError: 'lotSlug' is undefined
```

I think it should be `brief['lotSlug']`, as the manifest gets filtered by `brief` in the buyer frontend before display

https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/a54b9a071525ddcacac071973d3be65ed96c2b0d/app/main/views/marketplace.py#L183